app/ui-react/syndesis/src/modules/data/pages/VirtualizationSqlClientPage.tsx

```diff
@@ -40,65 +41,274 @@ export class VirtualizationSqlClientPage extends React.Component<
+                                      }
+                                      dashboardHref={resolvers.dashboard.root()}
+                                      dashboardString={t('shared:Home')}
+                                      dataHref={resolvers.data.root()}
+                                      dataString={t('shared:Virtualizations')}
+                                      i18nViews={t('virtualization.views')}
+                                      i18nCancelText={t('shared:Cancel')}
+                                      i18nDelete={t('shared:Delete')}
+                                      i18nDeleteModalMessage={t(
+                                        'virtualization.deleteModalMessage',
+                                        {
+                                          name: virtualization.keng__id,
+                                        }
+                                      )}
+                                      i18nDeleteModalTitle={t(
+                                        'virtualization.deleteModalTitle'
+                                      )}
+                                      // i18nDraft={t('shared:Draft')}
+                                      // i18nEdit={t('shared:Edit')}
+                                      // i18nEditTip={t(
+                                      //  'virtualization.editDataVirtualizationTip'
+                                      // )}
+                                      // i18nError={t('shared:Error')}
+                                      /* TD-636: Commented out for TP
+                                         i18nExport={t('shared:Export')} */
+                                      i18nPublish={t('shared:Publish')}
+                                      // i18nPublished={t(
+                                      //  'virtualization.publishedDataVirtualization'
+                                      // )}
+                                      i18nUnpublish={t('shared:Unpublish')}
+                                      i18nUnpublishModalMessage={t(
+                                        'virtualization.unpublishModalMessage',
+                                        {
+                                          name: virtualization.keng__id,
+                                        }
+                                      )}
+                                      i18nUnpublishModalTitle={t(
+                                        'virtualization.unpublishModalTitle'
+                                      )}
+                                      onDelete={handleDelete}
+                                      /* TD-636: Commented out for TP
+                                         onExport={this.handleExportVirtualization} */
+                                      onUnpublish={handleUnpublish}
+                                      onPublish={handlePublish}
+                                      // publishingLogUrl={
```

Comment:
If not needed, please remove the commented out sections.

Reply:
Removed and added TODOs